.flake8

```diff
@@ -45,15 +45,4 @@ ignore=
     # Missing type annotation for **kwargs
     ANN003
 
-exclude =
-    .git,
-    .idea,
-    __pycache__,
-    demo/*,
-    tests/*,
-    venv,
-    manage.py,
-    .venv
```

Comment:
Why did you remove this?

Reply:
because its not necessary. flake8 runs only against python files and i needed to run it against the stuff in tests.

Reply:
I have my virtual environments in my project directory, and venv's are propped full of python files. Think it would be better to only remove the `tests/*` line and keep everything else as it was.

Reply:
pre-commit doesnt run against git ignored files...

as you wish though.